src/backend/Dockerfile

```diff
@@ -30,7 +30,7 @@ WORKDIR /dist
 RUN cp /build/main .
 
 # Build a small image
-FROM alpine
+FROM alpine:3.7
```

Comment:
should we perhaps be using the latest version?

Reply:
Good call, moving to `:latest` for now.